README.md

```diff
@@ -1 +1,3 @@
-# docs
\ No newline at end of file
+# Public team documents
+
+A first public repository
```

Comment:
why are the red dots?

Reply:
DId you mean red `docs`? Red are the lines that were removed. Green is lines that were added. Even if one character is replaced in the line, the entire line becomes red, and is getting replaced with a new line which is the same but one char. That's how git works. But also this is very useful to visually see the difference between what was there, and the new change